Submissions/aaditkamat.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Aadit Rahul Kamat",
+  "img": "https://avatars.githubusercontent.com/u/30969577?v=4",
+  "email": "plunges_abiotic.0q@icloud.com",
+  "links": {
+    "website": "https://aaditkamat.com",
+    "linkedin": "https://www.linkedin.com/in/aaditkamat",
+    "github": "https://github.com/aaditkamat"
+  },
+  "jobTitle": "Unemployed (Master's Student at the University of Florida)",
```

Comment:
Do you really want to say that you are unemployed?  I think you're selling yourself short here, if you are a student.

Reply:
Hey @sophiabrandt, I think I based it off the template and I couldn't say that I was employed because I'm a student, even though I'm working part time. You are right though, I could certainly modify this portion to emphasize that I'm a student. Thanks for the input!